src/components/tag/index.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from "classnames";
+import { Span } from "../text";
+import s from "./Tag.module.css";
+
+type TagColor =
+  | "brand"
+  | "blue"
+  | "green"
+  | "yellow"
+  | "purple"
+  | "red"
+  | "orange"
+  | "teal"
+  | "gray"
+  | "white";
+
+export interface TagProps {
+  name: string;
+  color?: TagColor;
+}
+
+export default function Tag({ name, color = "brand" }: TagProps) {
```

Comment:
I probably wouldn't make Tag a distinct component; but just have it be embedded inside of TagList.

I can't imagine us ever using a tag outside of the context of a list.

This would also allow us to use the `li` inside of the list as our Tag element and we could skip the extra div.

Reply:
Makes sense, have moved to one component 🙏